environment.yml

```
@@ -80,6 +82,9 @@ dependencies:
   - plotly~=5.18.0
   - seaborn~=0.13.2 # statistical plotting with matplotlib
 
+  # Machine Learning packages
```

Consider adding `py-xgboost` (CPU build?) here for https://github.com/geo-smart/website-2024/pull/7.

```suggestion
  # Machine Learning packages
  - py-xgboost~=2.1.1=cpu*
```